personalwebsite/projects/templatetags/project_tags.py

```diff
@@ -0,0 +1,28 @@
+from django import template
+
+register = template.Library()
```

Comment:
We already have a generic tag available for all the projects. I don't understand what is the purpose of creating a second set of tags.